packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -154,13 +154,21 @@ information, see the States Language spec.
 
 ### Task
 
-A `Task` represents some work that needs to be done. It takes a `resource`
-property that is either a Lambda `Function` or a Step Functions `Activity`
-(A Lambda Function runs your task's code on AWS Lambda, whereas an `Activity`
-is used to run your task's code on an arbitrary compute fleet you manage).
+A `Task` represents some work that needs to be done. There are specific
+subclasses of `Task` which represent integrations with other AWS services
+that Step Functions supports. For example:
+
+* `lambda.FunctionTask` -- call a Lambda Function
+* `stepfunctions.ActivityTask` -- start an Activity (Activities represent a work
+  queue that you poll on a compute fleet you manage yourself)
+* `sns.PublishTask` -- publish a message to an SNS topic
+* `sqs.SendMessageTask` -- send a message to an SQS queue
+* `ecs.FargateRunTask`/`ecs.Ec2RunTask` -- run a container task
```

Comment:
I rather these would have a common prefix, so maybe (along with my comment on "Task" above): `RunStepFunctionsTaskOnFargate` and `RunStepFunctionsTaskOnEc2`